### DISCUSSION TITLE
HIDDEN FILES

### DISCUSSION THREAD

**Discussion by theologyco** [iterate-ch/cyberduck#15806] [Q&A]
Hidden files??  on upload command, cyberduck (latest update) does not show all the files in the folder I want to upload although I know they are there (theuy show in the finder on Mac OS 12.7.4

**Comment by dkocher** [comment]
Choose `⌘⇧.` to toggle files starting with a dot to show in Finder.app.

**Comment by theologyco** [comment]
Thankyou David,

That did not work but I replaced the file from backup and now cyberduck reads it all
